highfive/newpr.py

```diff
@@ -329,7 +329,15 @@ def new_pr(payload, user, token):
     diff = api_req("GET", payload["pull_request"]["diff_url"])['body']
 
     msg = payload["pull_request"]['body']
+    title = payload["pull_request"]['title']
+
+    
+
+    if re.search("rollup", title, re.IGNORECASE) and owner is "rust-lang":
```

Comment:
Could this be too broad? Should we only check that the string starts with "rollup"?

Reply:
1. Yes this is extremely broad. It should only match the regex `/^Rollup of [0-9]+ pull requests$/` and nothing more.
2. `owner is "rust-lang"`, what? Use `==` for string comparison or [you may get surprising results](https://stackoverflow.com/questions/13782451/why-is-this-string-comparison-returning-false).